container/nginx/njs/ipfs-response.js

```diff
@@ -0,0 +1,27 @@
+function setHeaders(req) {
+  const format = req.variables.format_final;
+  const filename = req.args.filename;
+
+  const pathSegments = req.uri.split("/");
+  const cid = pathSegments[2];
+  const filePathSegments = pathSegments.slice(3);
+  const lastSegment = filePathSegments[filePathSegments.length - 1];
+
+  let name;
```

Comment:
Is it possible for us to enter this filter and return an undefined value for `name`?

Reply:
ya, if there's no filename or format, the L1 will return a 501 and there'll be a blank value in the header. Addressed with https://github.com/filecoin-saturn/L1-node/pull/449/commits/c8e8eab73edfaaa0d08af605e106f9d3c2616a0b